Dockerfile

```diff
@@ -1,49 +1,50 @@
-FROM	debian:wheezy
-
+FROM	   node:6.7.0-slim
```

Comment:
> Did you have something in mind

Nothing in particular, just wanted to check if we had anything that required it.